app/helpers/application_helper.rb

```diff
@@ -23,6 +23,10 @@ def markdown_to_html(raw)
     MarkdownHandler.render(raw)
   end
 
+  def password_errors?(user)
+    user.errors.keys.grep(/password/).any?
```

Comment:
would `user.errors.keys.include?('password')` do the trick?

Reply:
Fair call - but the regex will catch errors on `password_confirmation` and `current_password` as well, and I think that's what we need.